src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1939,6 +1939,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     -->
   <PropertyGroup>
     <ResolveAssemblyReferencesDependsOn>
+      ResolveProjectReferences;
+      FindInvalidProjectReferences;
+      PrepareProjectReferences;
```

Comment:
I don't understand why PrepareProjectReferencs is listed after "ResolveProjectReferences" - it's a dependency of ResolveProjectReferences.

Reply:
I'd also rather avoid listing it redundantly.